src/strategies/radicle-funding-nft/examples.json

```
@@ -0,0 +1,18 @@
+[
+  {
+    "name": "Example query",
+    "strategy": {
+      "name": "radicle-funding-nft",
+      "params": {
+        "address": "0x6b175474e89094c44da98b954eedeac495271d0f",
```

Can you make this work with checksummed address too?
```suggestion
        "address": "0x6B175474E89094C44Da98b954EedeAC495271d0F",
```